mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/ocs/OCSConstants.java

```diff
@@ -70,6 +70,13 @@
    */
   public static final String OCS_ORG_ID = "csorganization.id";
 
+  /**
+   * Key to obtain the OCS API version to be used, if not specified, the default api version for the mule version running will be used.
+   *
+   * @since 1.3.1 1.4.0
+   */
+  public static final String OCS_API_VERSION = "ocs.api.version";
+
```

Comment:
Why is the default api version value not defined here?

Reply:
The default api version value is something that is expected to change from one mule version to another. I think that if we expose that as part of the api we are making a commitment to not change it.